recipes/libnetcdf/bld.bat

```diff
@@ -0,0 +1,23 @@
+mkdir %SRC_DIR%\build
+cd %SRC_DIR%\build
+
+set HDF5_DIR=%LIBRARY_PREFIX%\cmake
+
+if "%ARCH%" == "64" (
+  set CMAKE_GENERATOR="Visual Studio 10 2010 Win64"
```

Comment:
We need something better here to build for the different MSVC versions.  You should be able to use one of the two examples from https://github.com/conda/conda-build/wiki/Windows-recipe-patterns#using-cmake-with-visual-studio